transformers/events/DOCUMENTATION.md

```diff
@@ -29,28 +31,27 @@ If the contract isn't already present in the environment you'll need to add it b
+        abi      = '<huge blob of json>'
         deployed = 8928180
     ```
  
-     * The address field is the contract address you searched for, as a string.
+    * The address field is the contract you searched for, as a string.
     * The abi can be found on the `contracts` tab in Etherscan (scroll down). You'll need the entire thing as a string, so it's best to use single quotes like above.
     * The deployed field is the block number of the transaction where the contract was deployed. That can be found on the Contract's page, by looking for the Contract Creator and clicking it's transaction hash. That will take you to the transaction where the it was deployed on.
 
 ### Create the Transformer
-
+    
 To create a custom transformer you will need to create a new `Transformer`
 struct with a `ToModels` method on it which converts a `core.EventLog` object
 (the raw, untransformed data) to an `event.InsertionModel` (the domain object).
 
-Note that for this step you may _not_ need to create the database
-migration, because to write the transformer you don't need to save the transformed
+Much of the time it is possible to complete this step without creating the database
+migration, because to write the transformer you don't actually save the transformed
 object to the database. You may be saving addresses, which
 go into the already existing address table and are referenced by a foreign key. 
```

Comment:
Hm, I'm confused by this line. I think every transformer writes to a new table? Even if the table is functionally just a join table of `log_id` + `address_id`, I think it's normally declared on a table matching the event name

Reply:
You don't have to do it _yet_ to make the test pass. You'll need it for the integration test.

Reply:
I think maybe I'm thrown off by the line about "You may be saving an address" - the reason you don't need a migration isn't because of the content of the table, it's because the unit tests for individual transformers don't exercise writing the new record to the DB (that only happens in integration tests), right?

Reply:
That's correct - also that's not new. It's how we've phrased it for a while.